From 8ab6564a572cfc29804a00e0acf296101a4d205c Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 18 Aug 2015 13:19:20 -0500
Subject: [PATCH 0240/5242] MLK-11381: arm: imx: dts: imx7d-12x12-lpddr3-arm2
 iomuxc_lpsr pad settings

commit  0669b4578c874adeddb4aaa0635f2dc2c3775a95 from
https://source.codeaurora.org/external/imx/linux-imx.git

- Add iomuxc lpsr pad settings, move pinctrl_usbotg1_vbus and
  pinctrl_usbotg2_vbus to iomuxc lpsr domain.

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
Fixed for LPSR rename in 4.14
Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |   28 ++++++++++++++-----------
 1 file changed, 16 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index dc2f903..8fb5d83 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -426,18 +426,6 @@
 			>;
 		};
 
-		pinctrl_usbotg1_vbus: usbotg1vbusgrp {
-			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO05__GPIO1_IO5    0x14
-			>;
-		};
-
-		pinctrl_usbotg2_vbus: usbotg2vbusgrp {
-			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO07__GPIO1_IO7    0x14
-			>;
-		};
-
 		pinctrl_usdhc1_1: usdhc1grp-1 {
 			fsl,pins = <
 				MX7D_PAD_SD1_CMD__SD1_CMD     0x17059
@@ -510,6 +498,22 @@
 	};
 };
 
+&iomuxc_lpsr {
+	imx7d-sdb {
+		pinctrl_usbotg1_vbus: usbotg1vbusgrp {
+			fsl,pins = <
+				MX7D_PAD_LPSR_GPIO1_IO05__GPIO1_IO5 	  0x14
+			>;
+		};
+
+		pinctrl_usbotg2_vbus: usbotg2vbusgrp {
+			fsl,pins = <
+				MX7D_PAD_LPSR_GPIO1_IO07__GPIO1_IO7    0x14
+			>;
+		};
+	};
+};
+
 &lcdif {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lcdif_dat
-- 
1.7.9.5

